Updates Team and Members Blueprint examples for ADO. (#1983)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/azure-devops/examples.md

## Mapping users and teams

In the following example you will ingest your Azure Devops users and teams to Port, you may use the following Port blueprint definitions and integration configuration:

<ProjectBlueprint/>

<UsersBlueprint/>

<TeamsBlueprint/>

<PortUsersAndTeamsAppConfig/>

:::tip To Learn more
- Refer to the [setup](azure-devops.md#setup) section to learn more about the integration configuration setup process.
- We leverage [JQ JSON processor](https://stedolan.github.io/jq/manual/) to map and transform Azure Devops objects to Port entities.
- Click [Here](https://learn.microsoft.com/en-us/rest/api/azure/devops/core/teams/get%20all%20teams?view=azure-devops-rest-7.1#team) for the Azure Devops team object structure.
- Click [Here](https://learn.microsoft.com/en-us/rest/api/azure/devops/core/teams/get-team-members-with-extended-properties?view=azure-devops-rest-7.1&tabs=HTTP#teammember) for the Azure Devops team member object structure.

:::

After creating the blueprints and saving the integration configuration, you will see new entities in Port matching your teams alongside their members.

> ## Mapping teams and members (Deprecated)

Reviewer: add it a warning /tip and refer to the `Mapping users and teams` section

After creating the blueprints and saving the integration configuration, you will see new entities in Port.

## Mapping users and teams

> In the following example you will ingest your Azure Devops users and teams to Port, you may use the following Port blueprint definitions and integration configuration:

Reviewer 2: ```suggestion
The following example blueprints and integration configurations demonstrate how to ingest Azure Devops users and teams into Port:
```

<ProjectBlueprint/>

<UsersBlueprint/>

<TeamsBlueprint/>

<PortUsersAndTeamsAppConfig/>

> :::tip To Learn more

Reviewer 2: ```suggestion
:::tip Learn more
```

> - Refer to the [setup](azure-devops.md#setup) section to learn more about the integration configuration setup process.
> - We leverage [JQ JSON processor](https://stedolan.github.io/jq/manual/) to map and transform Azure Devops objects to Port entities.
> - Click [Here](https://learn.microsoft.com/en-us/rest/api/azure/devops/core/teams/get%20all%20teams?view=azure-devops-rest-7.1#team) for the Azure Devops team object structure.
> - Click [Here](https://learn.microsoft.com/en-us/rest/api/azure/devops/core/teams/get-team-members-with-extended-properties?view=azure-devops-rest-7.1&tabs=HTTP#teammember) for the Azure Devops team member object structure.

Reviewer 2: ```suggestion
- Refer to the [setup](azure-devops.md#setup) section to learn more about the integration configuration setup process.
- Port leverages the [JQ JSON processor](https://stedolan.github.io/jq/manual/) to map and transform Azure Devops objects to Port entities.
- Click [Here](https://learn.microsoft.com/en-us/rest/api/azure/devops/core/teams/get%20all%20teams?view=azure-devops-rest-7.1#team) for the Azure Devops team object structure.
- Click [Here](https://learn.microsoft.com/en-us/rest/api/azure/devops/core/teams/get-team-members-with-extended-properties?view=azure-devops-rest-7.1&tabs=HTTP#teammember) for the Azure Devops team member object structure.
```
